src/helpers.php

```diff
@@ -4,18 +4,91 @@
 
 use function App\sage;
 use function App\config;
+use function App\asset_path;
+use \Wikimedia\RelPath;
 
 /**
  * Get Sage's dist path
  *
  * @param  string $path
  * @return string
  */
-function get_dist_path($path)
+function get_dist_path($path = '')
 {
-    return trailingslashit(config('theme.dir')) . trailingslashit('dist') . $path;
+	$dist_path = trailingslashit(config('theme.dir'));
+
+	if(!empty($path)) {
+		$dist_path .= trailingslashit('dist') . $path;
+	} else {
+		$dist_path .= 'dist';
+	}
+
+	return $dist_path;
 }
 
+
+/**
+ * Returns path to svg dir
+ *
+ * @param  string $icon
+ * @return string
+ */
+function get_svg_path() {
+	return apply_filters('bladesvg_image_path', get_dist_path('images/svg/icons'));
+}
+
+/**
+ * Returns absolute path to icon
+ *
+ * @param  string $icon
+ * @return string
+ */
+function get_icon_path_abs($icon) {
+	return trailingslashit(get_svg_path()) . $icon . '.svg';
+}
+
+/**
+ * Returns a path without file extension
+ *
+ * @param  string $icon
+ * @return string
+ */
+function strip_file_ext($path)
```

Comment:
As mentioned in [my comment here](https://github.com/Log1x/blade-svg-sage/issues/11#issuecomment-360426676), this function always seems to return the filename with a `./` at the beginning of it, which causes some non-fatal problems later with the path construction.

I'd suggest you change this function to:

```php
function strip_file_ext($path)
{
	return pathinfo($path, PATHINFO_FILENAME);
}
```

Even when you define several nested directories for your `bladesvg_image_path`, this function only ever receives the filename with no directories. As such, the code above is a bit cleaner (or maybe you don't even need this function and could call `pathinfo()` inline).

Reply:
Yes, I [fixed and inlined that part](https://github.com/strarsis/blade-svg-sage/blob/0ccb18c27adffd763099032067691459d8964a8d/src/helpers.php#L74).

Reply:
Great! Thanks :)